package-lock.json

```diff
@@ -1,9266 +1,8 @@
 {
   "name": "typescript-runtime-type-benchmarks",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
Oh, lockfile has changed. I think older Node & NPM does not support v2?

Reply:
sorry i haven't used npm for a long time, i switched to yarn v1 very quickly and now i mainly use pnpm.